test/gov/verify_agreements.sh

```diff
@@ -9,8 +9,43 @@ echo -e "${PREFIX} starting"
 function verifyAgreements {
         # Wait until there are agreements
         TARGET_NUM_AG=1
+        MUL_NUM_AG_1=0
+        MUL_NUM_AG_2=0
+        MUL_NUM_AG_3=0
+        MUL_NUM_AG_4=0
         if [ "${HZN_REG_TEST}" != "1" ]; then
             if [ "${PATTERN}" == "sall" ]; then
+              case ${MUL_AGENTS} in
+                1)
+                echo -e "${PREFIX} 1 extra agent"
+                MUL_NUM_AG_1=6
```

Comment:
Did you enumerate the totals this way so that each agent could have a different config and therefore a different number of agreements? otherwise this is really un-necessary

Reply:
Defer to future issue, this logic will be re-written.